.jenkins/library/vars/helpers.groovy

```diff
@@ -217,6 +217,28 @@ def azureContainerDownload(String container_name, String file_pattern, String st
     )
 }
 
+def dependenciesInstall(String dcap_url = "", String psw_url = "", String install_flags = "", String build_dir = "${WORKSPACE}") {
```

Comment:
Could you please write some documentation on using this helper function? Particularly, install_flags and how they differ as extra args for ansible and the script params for windows.

See https://www.infoworld.com/article/2074120/documenting-groovy-with-groovydoc.html for the syntax